src/game.zig

```diff
@@ -521,6 +521,7 @@ pub const World = struct { // MARK: World
 	milliTime: i64,
 	gameTime: Atomic(i64) = .init(0),
 	spawn: Vec3f = undefined,
+	connected: Atomic(bool) = .init(false),
```

Comment:
I don't think an atomic would even be enough, because it gives no guarantees on the ordering. (`conn` might be deinited before the value is written to the variable)
So better just leave it non-atomic so any data races could at least be captured by the thread sanitizer.